hr_expense_invoice/views/hr_expense_views.xml

```diff
@@ -1,111 +1,98 @@
+<?xml version="1.0" encoding="utf-8" ?>
 <odoo>
     <record id="hr_expense_view_form" model="ir.ui.view">
         <field name="name">hr.expense.form</field>
         <field name="model">hr.expense</field>
         <field name="inherit_id" ref="hr_expense.hr_expense_view_form" />
         <field name="arch" type="xml">
-            <field name="unit_amount" position="attributes">
-                <attribute name="force_save">1</attribute>
+            <field name="price_unit" position="attributes">
+                <attribute name="force_save">
+                    1
+                </attribute>
             </field>
             <label for="quantity" position="attributes">
-                <attribute
-                    name="attrs"
-                >{'invisible': [('invoice_id', '!=', False)]}</attribute>
+                <attribute name="invisible">invoice_id != False</attribute>
```

Comment:
`not invoice_id`

Same in similar cases.

Besides, it should be:

`<attribute name="invisible" add="not invoice_id" separator=" or " />`

Reply:
done but`invoice_id != False` is not the same as `not invoice_id`